dependencies/gmp/include/gmp.h

```diff
@@ -1,2329 +1,13 @@
-/* Definitions for GNU multiple precision functions.   -*- mode: c -*-
 
-Copyright 1991, 1993-1997, 1999-2016 Free Software Foundation, Inc.
+//TODO remove if cmake is ready
+#define __CRYPTID_GMP
 
-This file is part of the GNU MP Library.
+#if defined(__CRYPTID_GMP)
```

Comment:
In other parts of the library, we've always used `#ifdef`. For the sake of consistency, we should stick to that form in my opinion.